baiji/connection.py

```diff
@@ -119,12 +119,12 @@ def cp_r(self, dir_from, dir_to, parallel=False, **kwargs):
                 # Emulate `cp`, which copies the contents of the path.
                 # Get path relative to from_path
                 files_to_copy = [(f, path.join(dir_to, os.path.relpath(path.parse(f).path, from_path)))
-                                 for f in self.ls(dir_from, return_full_urls=True)]
+                                 for f in self.ls(dir_from, return_full_urls=True) if not f.endswith(path.sep)]
```

Comment:
Yeah, I just don't like `isdirlike` but I suppose it is a fair point. I'll change it.
Thanks